fecgov/fec-cms
# Add download headshot button to commissioner pages

## Summary (required)

- Resolves #6467
- Add `headshot_download` image field to commissioner pages Wagtail template
- Add "Download headshot" button on the page, if a`headshot_download` image has been uploaded
- Include associated migration file for model change
- Tested in Chrome, Firefox, Edge, Safari, Opera, Brave

### Required reviewers

one ux -OR-  one frontend

## Impacted areas of the application

Commissioner pages

## Screenshots

![download](https://github.com/user-attachments/assets/8ffd896b-3b68-42f7-b4b2-80eb97a18cb8)


## How to test
- checkout branch and  `./manage.py migrate`
- Edit Commissioner Cooksey's page `picture_download`  field and upload this photo (or any photo)  to your local:
 https://www.fec.gov/resources/cms-content/images/Cooksey.original.png
-  Publish or preview the page and click "Download headshot" button
- Remove the image to confirm that the button does not render if there is no image in that field
